### PR TITLE
fix #60962 uludagsozluk.com

### LINKED ISSUE #60962: uludagsozluk.com
### Issue URL (Ads)
 [https://www.uludagsozluk.com/e/9253609/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.uludagsozluk.com%2Fe%2F9253609%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (1440), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by stanislav-atr**
#60962

new rules #125-127 added, one rule modified, obsolete rules removed, grouped

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -122,6 +122,20 @@ aykutozdemir.com.tr##.theiaStickySidebar > #text-4
 aykutozdemir.com.tr##.theiaStickySidebar > #text-5
 aykutozdemir.com.tr##.theiaStickySidebar > #text-10
 7deniz.net###lock > div[style] > a[href][target="_blank"] > img
+uludagsozluk.com###overlay
+uludagsozluk.com##.popup
+uludagsozluk.com###main > a[href*="bit.ly"][target="_blank"]
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:127]: It's better to exchange this rule to the common `##a[href^="https://bit.ly/"] > img`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply TurkishFilter/sections/specific.txt:127]: Also we have a similar rule `kofdizi.com###overlay` for `#overlay`. Add a domain to them.

Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt:127]: Suggested rule will leave 20px gap in website layout wich could be considered as an ad leftover.
I am targeting <a> tag instead to go seamless.

Comment (zloyden) [reply TurkishFilter/sections/specific.txt:127]: Ok, then you can throw away the part `img` and sorte similar rules to one common.

**Comment by Alex-302** [comment]
I can't reproduce popup.

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**